pandas/compat/__init__.py

```diff
@@ -65,3 +65,16 @@ def is_platform_mac():
 
 def is_platform_32bit():
     return struct.calcsize("P") * 8 < 64
+
+
+def import_lzma():
+    import warnings
```

Comment:
Can put this import at the top.

Reply:
Will move `import warnings` to the top. Thanks!